### PR TITLE
Updates to service config, links, note on UI

### PR DESCRIPTION
**Pull request by TownLake**

### CODE REVIEW

File: products/argo-tunnel/src/content/create-tunnel/index.md [hunk 1]
```diff
@@ -70,4 +70,10 @@ $ cloudflared tunnel delete -f <NAME>
 
 This will cause those connections to be dropped.
 
-Deleting the tunnel also invalidates the credentials file associated with that Tunnel, meaning those connections can not be re-established again.
\ No newline at end of file
+Deleting the tunnel also invalidates the credentials file associated with that Tunnel, meaning those connections can not be re-established again.
+
+<Aside>
+
+Argo Tunnels created in this method do not currently display in the Traffic tab of the Cloudflare dashboard.
```
Comment (adamchalmers) [inline products/argo-tunnel/src/content/create-tunnel/index.md:77]: Can we add something about "we're working on a new UI for this?"

File: products/argo-tunnel/src/content/getting-started/installation.md [hunk 2]
```diff
@@ -67,19 +67,32 @@ The update will cause `cloudflared` to restart which would impact traffic curren
 
 We recommend this option if you are currently using Cloudflare's Load Balancer product with your Argo Tunnel deployment.
 
-1. Install a new instance of `cloudflared` and [create](/configuration/index) a new Argo Tunnel.
-2. Configure the instance to point traffic to the same service or URL as your current, active instance of `cloudflared`.
+1. Install a new instance of `cloudflared` and [create](/create-tunnel) a new Argo Tunnel.
+2. Configure the instance to point traffic to the same locally available service as your current, active instance of `cloudflared`.
```
Comment (adamchalmers) [inline products/argo-tunnel/src/content/getting-started/installation.md]: should this be "locally-available"? Here and elsewhere in the PR too.

File: products/argo-tunnel/src/content/create-tunnel/index.md (hunk 1 above)
Comment (adamchalmers) [inline products/argo-tunnel/src/content/create-tunnel/index.md]: Can we add something about "we're working on a new UI for this?"

**Review by adamchalmers — APPROVED** [review]
Approved with a few comments

File: products/argo-tunnel/src/content/getting-started/installation.md (hunk 2 above)
Comment (TownLake) [reply products/argo-tunnel/src/content/getting-started/installation.md]: Yeah good catch. thanks

**Pull request merged**